ros_gp_mpc/config/configuration_parameters.py

```diff
@@ -57,11 +57,18 @@ class ModelFitConfig:
     """
 
     # ## Dataset loading ## #
-    ds_name = "gazebo_dataset"
+    ds_name = "simplified_sim_dataset"
     ds_metadata = {
-        "gazebo": "default",
+        "noisy": True,
+        "drag": True,
+        "payload": False,
+        "motor_noise": True
     }
 
+    # ds_metadata = {
```

Comment:
remove commented code. If this is a useful setting that is intended to be 'commented in' for another experiment, then maybe add a comment indicating this